<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportMetadataDlg</class>
 <widget class="QDialog" name="ExportMetadataDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export cards</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Card set name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="card_set_name"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Author name:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="author_name"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Author email:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Date:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDateEdit" name="date"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Revision:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="revision">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="author_email"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="tags"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Tags:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Notes:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="notes"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ok_button">
         <property name="text">
          <string>&amp;OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>card_set_name</tabstop>
  <tabstop>author_name</tabstop>
  <tabstop>author_email</tabstop>
  <tabstop>tags</tabstop>
  <tabstop>date</tabstop>
  <tabstop>revision</tabstop>
  <tabstop>notes</tabstop>
  <tabstop>ok_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>ExportMetadataDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
